internal/models/agent.go

```diff
@@ -9,9 +9,9 @@ type VCenterRegistration struct {
 }
 
 type AgentLoginRequest struct {
-	UserID         string `json:"userid"`
+	UserID         string `json:"userId"`
```

Comment:
why have we changed the casing here, i think our rest-api request expects in small case ?

Reply:
this is the confirmation i was asking today. When I test, its expecting camel cased.

Reply:
let me check